nixos/doc/manual/from_md/development/writing-nixos-tests.section.xml

```diff
@@ -137,11 +137,10 @@ nixos-lib.runTest {
         </term>
         <listitem>
           <para>
-            By default, the Nix store in the VM is not writable. If you
-            enable this option, a writable union file system is mounted
-            on top of the Nix store to make it appear writable. This is
-            necessary for tests that run Nix operations that modify the
-            store.
+            By default, the Nix store in the VM is writable - a writable
+            union file system is mounted on top of the shared host Nix
+            store to make it appear writable. This is necessary for tests
+            that run Nix operations that modify the store.
```

Comment:
I am a bit confused why there are `nixos/doc/manual/development` and `nixos/doc/from_md/development` files containing the same text -- which one(s) should I change? (I have edited both here)

Reply:
Ah, CI made it clear -- I should edit `nixos/doc/manual/development` and then run the `md-to-db.sh` script